src/Company.rei

```diff
@@ -1,5 +1,5 @@
 let suffixes: unit => Js.Array.t(string);
-let companyName: (~format: int=?, unit) => string;
+let companyName: (~format: int=?, unit) => option(string);
```

Comment:
Can `companyName` return a nullable result?

Reply:
Because this is why we need to `getExn` 😄 I'm not sure that these changes are needed

This relates to every one of the changes in the function signatures

Reply:
If i take out `Belt.Option.getExn` it throw this error:
```sh
Error: This expression has type Js.String.t => array(Js.String.t)
       but an expression was expected of type option(string) => 'a
       Type Js.String.t = string is not compatible with type option(string) 
```

How do we handle that if we don't need `getExn`?

Reply:
It looks like a constraint that you added in this PR (making companyName return an option instead of a string)

Reply:
Am I missing something?

Reply:
You correct. I have removed it and tests pass